files/jquery.webui-popover/update.json

```diff
@@ -0,0 +1,9 @@
+{
+  "packageManager": "github",
+  "name": "webui-popover",
+  "repo": "sandywalker/webui-popover",
+  "files": {
+    "basePath":"dist",
+    "include": ["jquery.webui-popover.js", "jquery.webui-popover.min.js", "jquery.webui-popover.css","jquery.webui-popover.min.css"]
```

Comment:
The min files are enough for production. Can you please remove the unminified files?